src/v2/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -16,7 +16,29 @@ const AuctionsApp: React.FC<AuctionsAppProps> = ({ children }) => {
   return (
     <AppContainer>
       <AuctionsMeta />
-
+      <Box mt={3} ml={4}>
+        <Text variant="largeTitle">Auctions</Text>
+        <Text pt="15px" pb="15px">
+          Bid on thousands of new works every week in leading online auctions.
+        </Text>
+        <Link
+          href="https://www.artsy.net/how-auctions-work"
+          underlineBehavior="hover"
+        >
+          <Text variant="mediumText" pt="10px" pb="10px">
+            How to bid on Artsy{" "}
+            <ChevronIcon
+              title={null}
+              direction="right"
+              color="black"
+              height="15px"
+              width="14px"
+              top="3px"
+              left="3px"
+            />
+          </Text>
```

Comment:
I'm pretty sure we have a text + chevron link somewhere in force, but not totally sure offhand. Might want to ask #dev-help.

Reply:
I had posted in our #product-grow channel, and was directed to the Chevron styling that I am using here by Damon. I will post to dev help and see if anyone has anything better.

Reply:
Ah, if damon said this is the way to do it, its likely the way to do it! Mayeb one can extract this into a `ChevronLink` component and move it into `v2/Components` (in a follow-up PR).

Reply:
Hmmm I will break this out as a backlog ticket.